scholia/api.py

```diff
@@ -457,8 +457,8 @@ def entity_to_pages(entity):
     for statement in entity['claims'].get('P304', []):
         pages = statement['mainsnak']['datavalue']['value']
         return pages
-    else:
-        return ''
+
+    return ''
```

Comment:
The documentation to the function says that the function returns an empty string if the field is not set.